samples/Chat/src/app/utils/downloadHandler.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(attachment-download) */
+import { AttachmentMetadata, defaultAttachmentMenuAction } from '@azure/communication-react';
+
+// uncomment below line to use custom icons
+// import { WindowNew24Regular } from '@fluentui/react-icons';
+// import React from 'react';
+
+/* @conditional-compile-remove(attachment-download) */
+const attachmentDownloadOptions = {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  actionsForAttachment: (attachment: AttachmentMetadata) => {
+    // if download option is not defined, default download option will be used
+    // we are manually returning default download option here as an example
+    // but you don't have to do this if you want to use default download option
+    return [defaultAttachmentMenuAction];
+
+    // This is an example of how to add a custom action to the attachment menu
+    // You can make the list of menu actions to be dynamic based on the
+    // the attachment metadata such as the file extension, file name, etc, OR
+    // based on the message metadata such as the sender ID, message type, etc.
+    /*
```

Comment:
There is a maintenance cost to commented code; it will break if any of the API changes.

Reply:
that's true, but i feel like it's important to call out the usage here though since it's very easy to overlook how powerful this function can be.

For the API maintenance, I feel like this is also true for storybook right and i don't feel like we are doing anything too different here

Reply:
ok maybe let me create an alternate version of this function here instead of comments